emscripten/markerDecompress.c

```diff
@@ -96,24 +96,40 @@ void extractDataAndSave(const char* str, const char* name){
     int iset_final_index = 9;
 
     char *fsetInitialIndex = strstr(str, "\",\"fset\":\"");
+    if (fsetInitialIndex == NULL) {
```

Comment:
i added these check to get a better error message in case the .zft file is not correct. 
i would add WEBARKITLOGe but it will require to convert to c++. i will do in another PR.